misc-tools/dj_make_chroot.in

```diff
@@ -67,7 +67,8 @@ Options:
   -i <debs>    List of extra package names to install (comma separated).
   -r <debs>    List of extra package names to remove (comma separated).
   -l <debs>    List of local package files to install (comma separated).
-  -y           Force overwriting the chroot dir and downloading debootstrap.
+  -y           Force overwriting the chroot dir and installing debootstrap.
+  -f           Force the download of debootstrap even if it's already installed.
```

Comment:
What is the intended use of this extra option? Why would you want this if you already have it installed?

Reply:
The main reason was to allow someone to use a specific version of debootstrap. For example on Arch the packages are usually more up to date w.r.t. Debian or Ubuntu. Maybe I'm a bit too cautious, but it may be possible of a breaking change in debootstrap (or a bug in the upstream version) which can be safely avoided by using the pinned version provided by `-f`.

Reply:
Thanks for the explanation.